ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.ts

```diff
@@ -45,44 +35,33 @@ export function getNumFlowsFromEdge(edge: CustomEdgeModel): number {
     return numFlows;
 }
 
-export function getNumInternalFlows(
-    nodes: CustomNodeModel[],
-    edges: CustomEdgeModel[],
-    deploymentId: string
-): number {
-    const externalNodeIds = getExternalNodeIds(nodes);
-    const numInternalFlows =
-        edges?.reduce((acc, edge) => {
+export function getNumAnomalousInternalFlows(networkFlows: Flow[]) {
+    const numAnomalousInternalFlows =
+        networkFlows.reduce((acc, flow) => {
             if (
-                (edge.source === deploymentId && !externalNodeIds.includes(edge.target || '')) ||
-                (edge.target === deploymentId && !externalNodeIds.includes(edge.source || ''))
+                flow.isAnomalous &&
+                flow.type !== 'CIDR_BLOCK' &&
+                flow.type !== 'EXTERNAL_ENTITIES'
             ) {
-                const numFlows = getNumFlowsFromEdge(edge);
```

Comment:
I think since you have removed the only 2 places where this helper function `getNumFlowsFromEdge()` was used, you can delete the function itself--(it's in this same file, above^)--and its associated unit tests.